src/components/EditorFooter/index.tsx

```diff
@@ -38,26 +38,28 @@ const EditorFooter = () => {
       <Flex gap={8}>
         <FormControl display="flex" alignItems="center">
           <FormLabel
-            htmlFor="ruler-swith-btn"
+            htmlFor="ruler-switch-btn"
             mb="0"
             color="brand.primary"
             data-testid="ruler-label"
+            w="70px"
           >
-            {rulerLabel}
+            {ruler}
           </FormLabel>
           <Switch
-            id="ruler-swith-btn"
+            id="ruler-switch-btn"
             colorScheme="footerSwitch"
+            className="custom-switch"
             sx={{
               "span:first-child": {
                 bgColor: "brand.primary",
               },
```

Comment:
No first-child selector anymore. Please.

Reply:
changed the selector.